### PR TITLE
Fix docs link

### PR DESCRIPTION
**Pull request by ghost**
Missed this one earlier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ py.test botbot_plugins
 
 ## Contribute!
 
-We want you to contribute your own plugins to make BotBot.me better. Please [read the docs](https://github.com/metabrainz/botbot-plugins/blob/master/DOCS.md) and review our [contributing guidelines](https://github.com/metabrainz/botbot-plugins/blob/master/CONTRIBUTING.md) prior to getting started to ensure your plugin is accepted.
+We want you to contribute your own plugins to make BotBot.me better. Please [read the docs](https://brainzbot.readthedocs.io/) and review our [contributing guidelines](https://github.com/metabrainz/botbot-plugins/blob/master/CONTRIBUTING.md) prior to getting started to ensure your plugin is accepted.
```
Comment (LeoVerto) [inline README.md]: This should probably link to the plugin-specific docs instead.

**Review by LeoVerto — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Closing due to #9.

**Comment by LeoVerto** [comment]
> Closing due to #9.

Wait what? This is #9. Or do you mean metabrainz/brainzbot-core#9? That still doesn't fix this project's README :P

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Do you mean https://brainzbot.readthedocs.io/en/latest/plugins.html?

Comment (LeoVerto) [reply README.md]: Yes, exactly.

Comment (ghost) [reply README.md]: Fixed.

**Review by LeoVerto — APPROVED** [review]

**Pull request merged**